ote/src/cljs/ote/app/controller/service_search.cljs

```diff
@@ -71,6 +71,10 @@
                       operator))))
 
 
+(defn- operator-in-list? [operator list]
+  "Return nil if operator is not in given list"
```

Comment:
Docstring should be placed before parameters

Reply:
Fixed